doc/index.md

```diff
@@ -17,6 +17,9 @@ For Developers
 
 -   [Build Objective-C (iOS) WebSocket Clients](dev-objc/o_dev_objc.md)
 -   [Build Android and Java WebSocket Clients](dev-android/o_dev_android.md)
```

Comment:
It would be clearer with a **JMS Mobile Clients** heading, yes. But then you'd have to add **WebSocket Mobile Clients** too or it will look weird.